files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -394,3 +394,10 @@
   with_items:
     - login
     - sshd
+
+#
+# Enable CRA for external variants
+#
+- command: pam-auth-update --enable challenge-response
+  when:
+    - variant is regex("external-.*"):
```

Comment:
Is this a valid Ansible condition?

It looks like it's failing in the pre-push run you posted:
```
00:14:16.214  TASK [****-platform : command] **********************************************
00:14:16.214  task path: /opt/appliance-build/live-build/build/external-demo-aws/binary/var/lib/****-platform/ansible/10-****-platform/roles/****-platform/tasks/main.yml:401
00:14:16.214  fatal: [binary]: FAILED! => {
00:14:16.214      "msg": "The conditional check '{u'variant is regex(\"external-.*\")': None}' failed. The error was: unhashable type: 'AnsibleMapping'\n\nThe error appears to be in '/opt/appliance-build/live-build/build/external-demo-aws/binary/var/lib/****-platform/ansible/10-****-platform/roles/****-platform/tasks/main.yml': line 401, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n#\n- command: pam-auth-update --enable challenge-response\n  ^ here\n"
```

Reply:
There was a typo (a trailing colon) at the end of the line. Removed.

Reply:
appliance build of all variants with upgrade testing: http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/1668/
appliance build with upgrade test from 5.3.5.0: http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/1669/